### DISCUSSION TITLE
Activaton faild

### DISCUSSION THREAD

**Discussion by Yerramilli0429** [community/community#19160] [Copilot Conversations]
suddenly my GitHub copilot activation failed and in the output log it is mentioned user not autharised but till yesterday I used GitHub copilot to write programs

**Comment by rauzerfabiano** [comment]
Hi @Yerramilli0429  try this:

1. Go to https://github.com/github-copilot/tp_signup
2. Press the button [Continue without payment details.](https://github.com/github-copilot/signup/success)
3. In Suggestions matching public code; select Allow or Block.
4. Press Save and get started
5. Close or Reload the IDE or Code Editor

If that doesn't work, so just contacting [Github Support](https://support.github.com/) here.

They would usually respond within a few working days.

Let me know if this helped you.
